Uncertainty_Quantification/Deep500_recipe/recipe_UQ.py

```diff
@@ -0,0 +1,97 @@
+""" A recipe to reproduce the UQ of the following repo:
+    https://github.com/spcl/deep-weather
+
+"""
+import math
+
+import deep500 as d5
+# Using PyTorch as the framework
+import deep500.frameworks.pytorch as d5fw
+
+import torch
+from events import *
+from models import *
+from utils import *
+from losses import *
+import loader
+from scheduler import CyclicLRScheduler
+
+args = args_parser()
```

Comment:
that's weird, why would this be here? the recipe should be parameterized with everything. I'd replace with a fixed `dataclass` (as said above).